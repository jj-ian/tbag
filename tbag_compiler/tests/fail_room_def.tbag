/* Authors: Maria */
room {
        string place;
        void nonsense;
}

room Test {
        name = "Test";
        place = "here";
        nonsense = huh;
}

room Test2 {
        name = "Test2";
        place = "there";
        nonsense = wha;
}

Test <-> Test2;

start { Test }

true {
        endgame;
}
