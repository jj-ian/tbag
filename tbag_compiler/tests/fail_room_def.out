Fatal error: exception Failure("room defs didn't check out")
